packages/jest-cli/src/reporters/get_snapshot_summary.js

```diff
@@ -8,36 +8,43 @@
  */
 
 import type {SnapshotSummary} from 'types/TestResult';
+import type {GlobalConfig} from 'types/Config';
 
 import chalk from 'chalk';
-import {pluralize} from './utils';
+import {formatTestPath, pluralize} from './utils';
 
 const ARROW = ' \u203A ';
+const DOWN_ARROW = ' \u21B3 ';
+const DOT = ' \u2022 ';
 const FAIL_COLOR = chalk.bold.red;
+const OBSOLETE_COLOR = chalk.bold.yellow;
 const SNAPSHOT_ADDED = chalk.bold.green;
 const SNAPSHOT_NOTE = chalk.dim;
-const SNAPSHOT_REMOVED = chalk.bold.red;
+const SNAPSHOT_REMOVED = chalk.bold.green;
 const SNAPSHOT_SUMMARY = chalk.bold;
 const SNAPSHOT_UPDATED = chalk.bold.green;
 
 export default (
   snapshots: SnapshotSummary,
+  globalConfig: GlobalConfig,
   updateCommand: string,
 ): Array<string> => {
   const summary = [];
   summary.push(SNAPSHOT_SUMMARY('Snapshot Summary'));
   if (snapshots.added) {
     summary.push(
-      SNAPSHOT_ADDED(ARROW + pluralize('snapshot', snapshots.added)) +
-        ` written in ${pluralize('test suite', snapshots.filesAdded)}.`,
+      SNAPSHOT_ADDED(
+        ARROW + pluralize('snapshot', snapshots.added) + ' written ',
```

Comment:
can we move this extra space in `' written '` to the next phrase? Just like in line 45: https://github.com/facebook/jest/pull/6181/files#diff-ca77f197605867eb8c6fd8df31a8b042R45
Same change applies to line 57: https://github.com/facebook/jest/pull/6181/files#diff-ca77f197605867eb8c6fd8df31a8b042R57